src/index.js

```diff
@@ -1,60 +1,134 @@
+/* eslint-disable max-classes-per-file */
+
 import './style.css';
 import '@fortawesome/fontawesome-free/js/fontawesome';
 import '@fortawesome/fontawesome-free/js/solid';
 import '@fortawesome/fontawesome-free/js/regular';
 import '@fortawesome/fontawesome-free/js/brands';
 
+// variables
+
 const toDoList = document.querySelector('.to-do-list');
-const listItems = [
-  {
-    description: 'Feed the dog',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Make breakfast',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Get grocery',
-    completed: false,
-    index: 2,
-  },
-];
-
-for (let i = 0; i < listItems.length; i += 1) {
-  const newItem = document.createElement('li');
-  newItem.setAttribute('id', i);
-  newItem.innerHTML = `
-    <div>
-    <input type="checkbox" class="items-list"> 
-    <label> ${listItems[i].description}</label><br>
-    </div>
-    <i class="fa-solid fa-ellipsis-vertical"></i>
-    `;
-  toDoList.appendChild(newItem);
+
+// classes
+
+class List {
+  constructor(description, index) {
+    this.description = description;
+    this.completed = false;
+    this.index = index;
+  }
+}
+
+class Store {
+  static getLists = () => {
+    let Lists;
+    if (localStorage.getItem('Lists') === null) {
+      Lists = [];
+    } else {
+      Lists = JSON.parse(localStorage.getItem('Lists'));
+    }
+    return Lists;
+  }
+
+  static setLists = (list) => {
+    const Lists = this.getLists();
+    Lists.push(list);
+    localStorage.setItem('Lists', JSON.stringify(Lists));
+  }
+
+  static deleteLists = (deleted) => {
+    const Lists = this.getLists();
+    const filteredLists = Lists.filter(
+      (listItem) => listItem.index !== deleted,
+    );
+
+    // update index of list
+
+    filteredLists.forEach((filteredList, index) => {
+      filteredList.index = index + 1;
+    });
+
+    localStorage.setItem('Lists', JSON.stringify(filteredLists));
+  }
```

Comment:
- [ ] Great job done here but you should have these functionalities in a new JavaScript file. Let's refactor this